airflow/api/__init__.py

```diff
@@ -23,18 +23,14 @@
 from importlib import import_module
 
 from airflow.configuration import conf
-from airflow.exceptions import AirflowConfigException, AirflowException
+from airflow.exceptions import AirflowException
 
 log = logging.getLogger(__name__)
 
 
 def load_auth():
     """Load authentication backends."""
-    auth_backends = "airflow.api.auth.backend.default"
-    try:
-        auth_backends = conf.get("api", "auth_backends")
-    except AirflowConfigException:
-        pass
+    auth_backends = conf.get("api", "auth_backends")
```

Comment:
Are you sure that there is no chicken-egg problem as init time? I fear there was (?) a historic reason why there is a try/except block. If conf is not initialized at time of loading the module, would this break the start of airflow?